doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -29,91 +29,79 @@ An entity is represented by a a JSON object which contain the following properti
 * `id` for the entity ID, represented by a JSON string. If the entity has an `id` attribute itself it will be ignored.
 * `type` for the entity type, represented by a JSON string. If the entity has a `type` attribute itself it will be ignored.
   If the entity doesn't have a type, then this property is not included.
-* A collection of properties, one per entity attribute. Property values can be single values or they can include,
-  `type` and properties representing metadata as well. Property values which correspond to the primitive JSON datatypes
-  "string", "number" and "boolean" and with no metadata are rendered as regular JSON values. Property values
-  which specify a custom type or include metadata are rendered as JSON objects with the following properties:
-  * `value`: for the attribute value, rendered as a regular JSON value depending on the type. Attributes with an
-    user-defined informative types are rendered as JSON strings (TBD).
-  * `type`: a JSON string which represents the attribute type provided at attribute creation time.
+* A collection of properties, one per entity attribute. Properties representing attributes with no metadata or type are
+  rendered as regular JSON values. Properties representing attributes which specify a type or include metadata are rendered
+  as JSON objects with the following properties:
+  * `value`: for the attribute value, rendered as a regular JSON value (string, number or boolean).
```

Comment:
Although the way with dealing with objects/vector in attribute values has not been completely solved (see issue https://github.com/telefonicaid/fiware-orion/issues/1040), let's use that wording, mentioning also the array possibility.